Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -459,6 +463,16 @@ security credentials for pods.
   .. code-block:: bash
 
       $ kubectl exec app-pod -- curl -s -w "\n" -X GET http://169.254.169.254/latest/meta-data/
+      ami-id
```

Comment:
If we'll show the output (which I think is good), I think we should also fix the syntax highlighting for this block: `.. code-block:: shell-session`

Reply:
Done